tinkerbell/rufio/Chart.yaml

```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.2.1
+version: 0.2.2
```

Comment:
We don't really need this bump as the chart hasn't been released since 2.1 (only realized after pushing 🤷🏻).